_data/changelogs/component-checkbox.yml

```diff
@@ -2,6 +2,14 @@ title: Checkbox
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Updated radio and checkbox tile styling to have lighter borders.
+    summaryAdditional:
```

Comment:
Consider note related to what the accessibility issue it affects.

Reply:
I agree with James. I like when changelogs also address _why_ the change was made.

Reply:
Changed to `Updated radio and checkbox tiles to have lighter borders, reducing visual noise.`